docs/source/en/guides/cli.md

```diff
@@ -10,10 +10,18 @@ The `huggingface_hub` Python package comes with a built-in CLI called `huggingfa
 
 First of all, let's install the CLI:
 
+### Using pip
+
 ```
 >>> pip install -U "huggingface_hub[cli]"
 ```
 
+### Using pkgx
+
+```
+>>> pkgx install huggingface_cli
```

Comment:
Out of curiosity, would it be possible to have `pkgx install huggingface-cli` instead of `pkgx install huggingface_cli` ? This would make it more aligned with the existing.

Reply:
Oh, that's a type. It is supposed to be `huggingface-cli`. I'll correct it and implement your suggestions

Reply:
Ah, great!